linajea/tracking/track.py

```diff
@@ -20,10 +20,11 @@ def track(graph, parameters, selected_key,
 
             The candidate graph to extract tracks from
 
-        parameters (``TrackingParameters``)
+        config (``TrackingConfig``)
```

Comment:
I think this should be a SolveConfig, since we only every access config.solve in this function. For testing this function, it is a pain in the butt to create a whole valid TrackingConfig just to run the solver.